cscs-checks/prgenv/environ_check.py

```diff
@@ -90,13 +90,11 @@ def __init__(self):
 
         # FIXME: These modules should be fixed in later releases,
         # while gcc was fixed in 20.11
-
         cdt = osext.cray_cdt_version()
         if ((cdt and cdt <= '20.11' and
-             self.cray_module in ['cray-petsc-complex',
+             self.cray_module in {'cray-petsc-complex',
                                   'cray-petsc-complex-64',
-                                  'cudatoolkit']) or
-            (cdt and cdt < '20.11' and self.cray_module == 'gcc')):
```

Comment:
Since we are removing the gcc check I think we should also update the FIXME comment above. Remove the last part `# while gcc was fixed in 20.11`